src/lightning_lite/strategies/registry.py

```diff
@@ -112,11 +111,8 @@ def __str__(self) -> str:
         return "Registered Strategies: {}".format(", ".join(self.keys()))
 
 
-StrategyRegistry = _StrategyRegistry()
-
-
-def call_register_strategies(base_module: str) -> None:
+def call_register_strategies(registry: _StrategyRegistry, base_module: str) -> None:
```

Comment:
This is a breaking change. But I don't think this should have been exposed anyways. So if you are going to add an argument, I would also prefix it

Reply:
Yes